server/util/cosmos.js

```diff
@@ -195,16 +94,13 @@ export async function sendCoins(signer, targets) {
   });
 
   do {
-  /* eslint-disable no-await-in-loop */
+    /* eslint-disable no-await-in-loop */
     retry = false;
-    signedTx = signTransaction({
-      signer,
-      accNum,
-      stdTx,
-      sequence: pendingCount,
-    });
     try {
-      txHash = await sendTransaction(signedTx);
+      const { transactionHash } = await signingClient.signAndBroadcast(
+        delegatorAddress, msg, fee, memo,
+      );
```

Comment:
Please use `pendingCount` as `sequence` here, we want to be able to queue up multiple transactions at a time with correct sequence